app/models/invitation.rb

```diff
@@ -0,0 +1,22 @@
+class Invitation < ActiveRecord::Base
+
+	belongs_to :sender, :class_name => 'User'
+	has_one :recipient, :class_name => 'User'
```

Comment:
Please use 'new' style for hashes. Homework - why class name has to be in apostrophes?